### ISSUE TITLE
`KeyError: 'requestPath'` when used with API Gateway

### ISSUE THREAD

**Issue by nineinchnick** [logandk/serverless-wsgi#188]
After upgrading to 2.0 I started getting these errors:
```
[ERROR] KeyError: 'requestPath'
Traceback (most recent call last):
  File "/var/task/wsgi_handler.py", line 106, in handler
    return serverless_wsgi.handle_request(wsgi_app, event, context)
  File "/var/task/serverless_wsgi.py", line 173, in handle_request
    return handle_lambda_integration(app, event, context)
  File "/var/task/serverless_wsgi.py", line 295, in handle_lambda_integration
    path_info = event["requestPath"]
```

The docs for API Gateway Lambda integration doesn't mention `requestPath` anywhere: https://docs.aws.amazon.com/lambda/latest/dg/services-apigateway.html 

Can this be related to changes in this condition? https://github.com/logandk/serverless-wsgi/commit/d7a6ca146f51dba67436ad684ee11f164d59e808#diff-73eb20b9aafd70929ffc3487ee8a7cff8dedce9a6dc588e2cef517137836a82cL171

My `serverless.yaml` file:
```
service: xxx

plugins:
  - serverless-python-requirements
  - serverless-wsgi
  - serverless-domain-manager
  - serverless-api-gateway-throttling
  - serverless-apigw-binary
custom:
  stage: ${opt:stage}
  wsgi:
    app: api.app
    packRequirements: false
    pythonBin: python3
  pythonRequirements:
    zip: true
    pythonBin: python3
  domains:
    prod: xxx
  customDomain:
    domainName: ${self:custom.domains.${opt:stage}}
    basePath: ''
    stage: ${self:provider.stage}
    createRoute53Record: true
  apiGatewayThrottling:
    maxRequestsPerSecond: 5
    maxConcurrentRequests: 10
  apigwBinary:
    types:
      - 'text/plain'
      - 'application/x-ndjson'
      - 'application/x-jsonlines'

# exclude packages from the build before sending to AWS
package:
  exclude:
    - node_modules/**
    - venv/**
    - .pytest_cache/**
    - .serverless/**
    - tests/**

provider:
  name: aws
  runtime: python3.8
  region: us-east-2
  environment:
    ENV: ${self:custom.stage}

functions:
  app:
    handler: wsgi_handler.handler
    events:
      - http: ANY /
      - http: 'ANY /{proxy+}'
```

**Comment by gabrielleandro0801** [comment]
We're having the same problem here..
We deployed our API yesterday and we had no problems, but when we did it today around 3 hours after you commented we had exactly the same message

**Comment by logandk** [comment]
It's from #184 and #185 which was merged for the 2.0.0 release. I'll revert and release 2.0.1.

**Comment by gabrielleandro0801** [comment]
Thank you, man!
The released fixed our bug!
